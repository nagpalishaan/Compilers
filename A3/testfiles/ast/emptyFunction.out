Program
  List
    Function
      IdDecl ID="a"
      ParamList
        List
      List
