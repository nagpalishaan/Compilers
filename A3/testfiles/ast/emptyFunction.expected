Program
  List
    Function
      IdDecl ID="a"
      ParamList
        List
      StmtBlock
        List
